pkg/process/util/containers/containers.go

```diff
@@ -189,43 +188,6 @@ func (p *containerProvider) GetContainers(cacheValidity time.Duration, previousC
 	return processContainers, rateStats, pidToCid, nil
 }
 
-// GetPidToCid returns containers found on the machine
-func (p *containerProvider) GetPidToCid(cacheValidity time.Duration) map[int]string {
```

Comment:
Curious why this method was removed. 

I remember it is used in several parts of the codebase (example in the containerd workloadmeta collector)

Reply:
Merge issue, fixed, sorry!